### DISCUSSION TITLE
How do I connect to a panetscale db with navicat?

### DISCUSSION THREAD

**Discussion by x-orange474** [planetscale/discussion#244] [Q&A] [answered]
Hi,
When I connected to a panetscale db with navicat15, I run into difficulties with ssl. I looked through the documentation and didn't know how to fix it.
![error](https://user-images.githubusercontent.com/76637977/183085757-9a0c464a-557e-442d-a987-2de5d5b86a0d.PNG)

Any help would be appreciated! If I am not providing enough information please let me know. Thank you

**Comment by fooforge** [comment, marked as answer]
@x-orange474 I am fairly certain you just need to uncheck _Use authentication_ and keep _Use SSL_ checked to be able to connect. The authentication fields enable you to configure SSL/TLS Client Authentication, but to be able to connect securely to PlanetScale all that is needed is for your client to validate our server's certificate. Client certificates are not needed.

I was able to connect this way in a trial version of Navicat 16 from my Mac.

**Reply by kindredzhang** [reply]
thank very much.

**Comment by adk23333** [comment]
oh, only navicat 16 can connect it.

**Reply by kindredzhang** [reply]
https://www.nite07.com/navicat/
